src/value/borrowed.rs

```diff
@@ -325,16 +325,9 @@ impl<'val> Element for BorrowedElement<'val> {
         }
     }
 
-    fn as_blob(&self) -> Option<&[u8]> {
+    fn as_bytes(&self) -> Option<&[u8]> {
         match &self.value {
-            BorrowedValue::Blob(bytes) => Some(*bytes),
-            _ => None,
-        }
-    }
-
-    fn as_clob(&self) -> Option<&[u8]> {
-        match &self.value {
-            BorrowedValue::Clob(bytes) => Some(*bytes),
+            BorrowedValue::Blob(bytes) | BorrowedValue::Clob(bytes) => Some(*bytes),
```

Comment:
Same here--changes to both `clob`/`blob` accessing data as `&[u8]` through a single API.